docs/framework/windows-workflow-foundation/persistence-database-schema.md

```diff
@@ -27,7 +27,7 @@ This topic describes the public views supported by the SQL Workflow Instance Sto
 |ExecutionStatus|Nvarchar(450)|Indicates the current execution state of the Workflow. Possible states include **Executing**, **Idle**, **Closed**.|  
 |IsInitialized|Bit|Indicates whether the workflow instance has been initialized. An initialized workflow instance is a workflow instance that has been persisted at least once.|  
 |IsSuspended|Bit|Indicates whether the workflow instance has been suspended.|  
-|IsCompleted|Bit|Indicates whether the Workflow Instance has finished executing. **Note:**  Iif the **InstanceCompletionAction** property is set to **DeleteAll**, the instances are removed from the view upon completion.|  
+|IsCompleted|Bit|Indicates whether the Workflow Instance has finished executing. **Note:**  If the **InstanceCompletionAction** property is set to **DeleteAll**, the instances are removed from the view upon completion.|  
```

Comment:
I think this was meant to be the "iif" shorthand for "if and only if". If so, we should spell it out:

```suggestion
|IsCompleted|Bit|Indicates whether the Workflow Instance has finished executing. **Note:**  If and only if the **InstanceCompletionAction** property is set to **DeleteAll**, the instances are removed from the view upon completion.|  
```